openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,62 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' not in data:
+            return
+        if 'load' in data['resources']:
+            extra_values = {
+                'load_1': data['resources']['load'][0],
+                'load_5': data['resources']['load'][1],
+                'load_15': data['resources']['load'][2],
+            }
+            metric, created = Metric._get_or_create(
+                object_id=pk,
+                content_type=ct,
+                key='load',
+                field_name='load',
+                name='Resources: Load',
+            )
+            metric.write(
+                data['resources']['load'][1] / data['resources']['cpus'],
```

Comment:
Don't worry about the percentage. Just save the raw data and we will make the transformations later for the user.

Just write `data['resources']['load'][0]`

Reply:
Saving the number of cores, isn't helpful I guess since it's a constant